lib/srv/alpnproxy/proxy.go

```diff
@@ -436,7 +439,7 @@ func (p *Proxy) databaseHandlerWithTLSTermination(ctx context.Context, conn net.
 
 func isDBTLSProtocol(protocol common.Protocol) bool {
 	switch protocol {
-	case common.ProtocolMongoDB, common.ProtocolRedisDB:
+	case common.ProtocolMongoDB, common.ProtocolRedisDB, common.ProtocolSQLServer:
 		return true
 	default:
 		return false
```

Comment:
nit: not sure if it will be better to have this function in `common/protocols.go` so it's easier to update it when adding a new protocol.

Reply:
Yes, seems to be reasonable to move it to common where protocols are added.